reviews/sandbox/2022/2022-Confidential-Containers-annual.md

```diff
@@ -0,0 +1,155 @@
+- [@magowan](https://github.com/magowan), James Magowan, IBM
+- [@peterzcst](https://github.com/peterzcst), Peter Zhu, Intel
+- [@sameo](https://github.com/sameo), Samuel Ortiz, Rivos
+
+## Adoption
+
+> **Note**
+> From CNCF: What do you know about adoption, and how has this changed since your last review / since you joined Sandbox? If you can list companies that are adopters of your project, please do so._
+
+Cloud Providers (Microsoft, Alibaba, IBM) are pusuring offerings which will
+include Confidential Containers. They are actively engaged in the community.
+
+At recent KubeCon EU, Microsoft invited early adopters to sign up to try out
+Confidential Containers in Azure.
+
+## Project Goals
+
+### Performance
+
+> **Note**
+> From CNCF: _How has the project performed against its goals since the last review?_
+
+This is the first annual Review for Confidential Containers, so the comparison
+baseline is the the moment when Confidential Containers was accepted as a CNCF
+Sandbox project, i.e. March 2022.
+
+Since then, the Confidential Containers project published 6
+[releases](https://github.com/confidential-containers/documentation/tree/main/releases)
+by following a time-based, 6 weeks period release cadence.
+
+Starting from the initial release, the Confidential Containers delivered on its
+initial and major goal of enabling seamless use and deployment of unmodified
+containers through Kubernetes, across multiple TEEs and hardware platforms.
+
+The project first 6 releases brought the following major key features and
+enhancements:
+
+- A stabilized, enhanced and wider CI that is key to the project's fast release
+  cadence.
+- A hardware backed CI, i.e. a CI that runs test on actual confidential
+  computing hardware instances like e.g. Intel TDX, AMD SEV and Intel SGX.
+- A Kubernetes Operartor for easily deploying the Confidential Containers
+  runtime, stack and payload on Kubernetes clusters.
+- Support for process-based TEEs like e.g. SGX.
+- Technical debt reduction, in particular the dependencies on `umoci` and
+  `skopeo` are gone.
+- A modular, [RATS-compliant](https://datatracker.ietf.org/doc/rfc9334/) Key
+  Broker Service (KBS) that implements a full remote attestation service. The
+  Confidential Container KBS provides its own attestation service but also
+  supports external ones like Intel Amber.
+
+### Current Goals
+
+> **Note**
+> From CNCF: _What are the current goals of the project? For example, are you working on major new features? Or are you concentrating on adoption or documentation?_
+
+At a high level, the current project focus is on stability and reliability
+increase, performance improvements and customer adoption expansion.
+
+This translates into the following goals:
+
+1. CI/CD improvements
+1. Host-Based container images pulling
+1. Extended attestation model
+1. Policy and secure container configuration
+1. Kata containers upstreaming
+1. Use cases driven implementation
```

Comment:
@sameo can we change "Use cases driven implementation" to:

Use cases driven implementation replacing feature driven development